substrate/frame/revive/fixtures/contracts/call_data_load.rs

```diff
@@ -0,0 +1,44 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This uses the call data load API to first the first input byte.
```

Comment:
```suggestion
//! This uses the call data load API to get the first input byte.
```

Reply:
wouldn't be easier to just unit test with a single call_data_load statement?

Reply:
But we have to read out the offset anyways; I think it makes no difference what API method we use for that?